pytorch_blade/benchmark/TorchBench/test_torch_bench.sh

```diff
@@ -30,32 +30,43 @@ python3 -m pip install -U numpy
 pushd $script_dir # pytorch_blade/benchmark/TorchBench
 ln -s $benchmark_repo_dir torchbenchmark
 
-# benchmark 
-config_file=blade_$1_$2.yaml
-bench_target=$2
+# benchmark
 # setup benchmark env
 export DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE=true \
     DISC_CPU_LARGE_CONCAT_NUM_OPERANDS=4 DISC_CPU_ENABLE_EAGER_TRANSPOSE_FUSION=1 \
     OMP_NUM_THREADS=1 TORCHBENCH_ATOL=1e-2 TORCHBENCH_RTOL=1e-2
-if [ $2 == "cpu" ];then
+
+config_file=blade_$1_$2.yaml
+bench_target=$2
+
+if [ $1 == "cpu" ]
+then
     # 4 cores
     export GOMP_CPU_AFFINITY="2-5"
+    results=(eval-cpu-fp32)
+else
+    results=(eval-cuda-fp32 eval-cuda-fp16)
 fi
 python3 torchbenchmark/.github/scripts/run-config.py -c $config_file -b ./torchbenchmark/ --output-dir .
-# results
 
+# results
+date_str=$(date '+%Y%m%d-%H')
+oss_link=https://bladedisc-ci.oss-cn-hongkong.aliyuncs.com
+oss_dir=oss://bladedisc-ci/TorchBench/${bench_target}/${date_str}
 
+for result in ${results[@]}
+do
+    cat ${result}/summary.csv
+    curl ${oss_link}/TorchBench/baseline/${result}_${bench_target}.csv -o $result.csv
```

Comment:
Will the baseline be updated manually?

Reply:
Yes, I will update it when some performance promoted.